engine.py

```diff
@@ -196,7 +213,7 @@ def _define_qt_base(self):
         """
         check for pyside then pyqt
         """
-        
+
```

Comment:
All this logic is already handled by core. You should simply call the base class and if Qt is not available you return the proxy.